src/getting_started.md

```diff
@@ -3,47 +3,49 @@
 Instead of re-hashing content which has been written about before, we instead provide a curated list of topics that we'll assume familiarty with from the reader.
 
 <!-- Should probably trim down the links to only the absolute necessary to not overwhelm
-the reader-->
+the reader, maybe could add indications as to what is required/optional reading?-->
 
 1. How does Ethereum work?
     1. [Ethereum Development Documentation](https://ethereum.org/en/developers/docs/)
-    1. [How does Ethereum work, anyway?](https://preethikasireddy.medium.com/how-does-ethereum-work-anyway-22d1df506369)
+    2. [How does Ethereum work, anyway?](https://preethikasireddy.medium.com/how-does-ethereum-work-anyway-22d1df506369)
```

Comment:
These should not be required, markdown auto orders numbered lists!

Reply:
Oh interesting TIL, will fix